slack_sdk/web/async_client.py

```diff
@@ -393,7 +393,7 @@ async def admin_conversations_rename(
 
     async def admin_conversations_search(self, **kwargs) -> AsyncSlackResponse:
         """Search for public or private channels in an Enterprise organization."""
-        return await self.api_call("admin.conversations.search", json=kwargs)
+        return await self.api_call("admin.conversations.search", params=kwargs)
```

Comment:
As I mentioned in the description, the `api_call` with the JSON request body is not compatible with the iterator by `SlackResponse`.